tests/functional/modules/test_zos_blockinfile_func.py

```diff
@@ -12,13 +12,12 @@
 # limitations under the License.
 
 from __future__ import absolute_import, division, print_function
-from ibm_zos_core.tests.helpers.dataset import get_tmp_ds_name
-from shellescape import quote
 import time
 import re
-import pytest
 import inspect
-import os
+from shellescape import quote
```

Comment:
Maybe I'm mis-remembering, but shouldn't this be "from shlex import quote"?  I thought shellescape was for older python.

Reply:
good catch opened issue #1649